fox/collectorfox/app.py

```diff
@@ -1,11 +1,11 @@
 import api
 from collector import collectorfox
-from transformation import RTXPipeline
+from transformation import LeRobotPipeline
 from annotation import TextAnnotator, ClipAnnotator
 
 if __name__ == '__main__':
-    collectorfox.register_transformation("cmu_stretch", "rtx", RTXPipeline(step_data_sample_col=["language_instruction", "observation"], step_data_sample_method="first"))
-    collectorfox.register_transformation("ucsd", "rtx", RTXPipeline(step_data_sample_col=["language_instruction", "observation"], step_data_sample_method="first"))
+    collectorfox.register_transformation("cmu_stretch", "rtx", LeRobotPipeline(step_data_sample_col=["language_instruction", "observation"], step_data_sample_method="first"))
+    collectorfox.register_transformation("ucsd", "rtx", LeRobotPipeline(step_data_sample_col=["language_instruction", "observation"], step_data_sample_method="first"))
     collectorfox.register_annotators("cmu_stretch", "language_instruction", "language_embedding", TextAnnotator("sentence-transformers/all-MiniLM-L6-v2"))
     collectorfox.register_annotators("ucsd", "language_instruction", "language_embedding", TextAnnotator("sentence-transformers/all-MiniLM-L6-v2"))
     collectorfox.register_annotators("ucsd", "observation", "clip_embedding", ClipAnnotator())
```

Comment:
Can we make them one universal dataset here? Let's call it fogx-universal-lerobot

Reply:
And should modify https://github.com/J-HowHuang/FogX-Store/blob/collector-lerobot/tests/prepare_data.py accordingly

Reply:
> Can we make them one universal dataset here? Let's call it fogx-universal-lerobot
Might be challenging. Each dataset has unique metadata for the episode level. But the steps level are all in the same schema now (in the LeRobot schema)



> And should modify https://github.com/J-HowHuang/FogX-Store/blob/collector-lerobot/tests/prepare_data.py accordingly

Will do that.

Reply:
No need to include all metadata fields, and we can decide which fields to include here. Or we can drop any episode level metadata, just sample fields such as language instruction and observation to episode level (need to make sure dataset without those fields should put null as the value.

Reply:
> No need to include all metadata fields, and we can decide which fields to include here. Or we can drop any episode level metadata, just sample fields such as language instruction and observation to episode level (need to make sure dataset without those fields should put null as the value.

The latter makes more sense to me. Will implement that.

Reply:
> Can we make them one universal dataset here? Let's call it fogx-universal-lerobot
Fix at [2415dbe](https://github.com/J-HowHuang/FogX-Store/pull/8/commits/2415dbe9aa37e666646cb3523a48529484035084)